README.md

```diff
@@ -54,11 +54,12 @@ class MyComponent extends Coroutine {
   }
 
   render() {
-    switch (this.state.data.status) {
+    const { data: { status, users, error } } = this.state
```

Comment:
I agree about improved readability because of destructuring. However, nested destructuring may be confusing for newcomers. Do you mind updating it to one level? Here is an example:

```javascript
const { status, users, error } = this.state.data;
```